### PR TITLE
v2.5.0

### PR DESCRIPTION
**Pull request by zachmullen**

### CODE REVIEW

**Comment by mgrauer** [comment]
@girder/developers 

Here's a list of PRs that have been merged since the last release and are not referenced in the CHANGELOG. I'd like to crowdsource updates to the CHANGELOG, so take a look here and if you have ownership or awareness of one of the listed PRs, please evaluate if it needs to be added to the CHANGELOG and if so add it in the appropriate section, then check the checkbox.

- [ ] #2467 from girder/s3-import-folder-button
- [ ] #2637 from girder/job-model-save-kwargs
- [ ] #2635 from nicholsn/master
- [ ] #2634 from girder/pytest-enable-plugins
- [ ] #2633 from girder/get-file-path
- [ ] #2632 from girder/fix-s3-assetstore-test
- [ ] #2631 from girder/webpack-define-env
- [ ] #2630 from girder/fix-pip-reinstall
- [ ] #2527 from girder/more-pytests
- [ ] #2526 from girder/sphinx-build-fix
- [ ] #2524 from girder/raise-request-exceptions-in-test-harness
- [ ] #2523 from girder/modal-dialog-max-height
- [ ] #2522 from girder/stream-to-logger-properties
- [ ] #2638 from girder/silence-expected-error-messages
- [ ] #2620 from girder/virtual-folders-plugin
- [ ] #2430 from girder/disable-registration
- [ ] #2493 from girder/harden-swagger-test
- [ ] #2531 from girder/pytest-dropdb-default
- [ ] #2496 from girder/fix-plugin-dependencies
- [ ] #2538 from girder/reqs
- [ ] #2621 from girder/reset-password
- [ ] #2622 from girder/verify-email-client
- [ ] #2462 from girder/paging-validation
- [ ] #2532 from girder/add-tox-documentation
- [ ] #2535 from girder/test-client-pythonpath
- [ ] #2537 from girder/pytest-keep-db
- [ ] #2629 from girder/fix-typo
- [ ] #2461 from girder/catch-missing-version-number
- [ ] #2487 from girder/cherrypy-12-response-timeout
- [ ] #2654 from girder/jquery-browser
- [ ] #2656 from girder/new-style-classes
- [ ] #2544 from girder/py-branch-coverage
- [ ] #2546 from girder/package-json
- [ ] #2541 from girder/pytest-strict-markers
- [ ] #2540 from girder/mongo-3.2
- [ ] #2542 from girder/flake8-docstring-workaround
- [ ] #2492 from girder/fix-pytest-girder-setup-py
- [ ] #2529 from girder/even-more-pytests
- [ ] #2528 from girder/tox-integration
- [ ] #2497 from girder/fix-api-prefix-doc-typo
- [ ] #2647 from girder/pydicom
- [ ] #2642 from girder/disable-event-thread-option
- [ ] #2662 from girder/ci-artifacts
- [ ] #2491 from girder/harden-routing-text
- [ ] #2557 from girder/npm-deps
- [ ] #2554 from girder/update-flake8-docstrings
- [ ] #2498 from girder/missing-assetstores
- [ ] #2086 from girder/treeview
- [ ] #2481 from girder/fix-terms-client-test
- [ ] #2483 from girder/task-test-locale
- [ ] #2355 from girder/misc
- [ ] #2562 from girder/swagger-zip-downloads
- [ ] #2560 from girder/tox-install
- [ ] #2565 from girder/bind-to-localhost-by-default
- [ ] #2564 from girder/ansible-upgrade-setuptools
- [ ] #2341 from girder/update-mailing-list-link
- [ ] #2347 from girder/view-for-search-results
- [ ] #2581 from girder/another-pytest-conversion
- [ ] #2583 from girder/revert-circle-workaround
- [ ] #2582 from girder/workaround-circle-mongo-bug
- [ ] #2585 from girder/upgrade-uglify-plugin
- [ ] #2584 from girder/empty-layout-no-pad
- [ ] #2586 from girder/mongomock-from-gh
- [ ] #2575 from girder/xss-2504
- [ ] #2577  from girder/sphinx-tox
- [ ] #2570 from girder/support-host-arg-to-girder-server
- [ ] #2571 from girder/log-on-upload
- [ ] #2572 from girder/min-python
- [ ] #2276 from girder/2232-update-docs-to-include-changelog
- [ ] #2578 from girder/item-href
- [ ] #2270 from girder/item_tasks_from_gw_describe
- [ ] #2668 from girder/remove-flake8-show-pep8
- [ ] #2669 from girder/subprocess-pip-install
- [ ] #2476 from girder/download-iter
- [ ] #2660 from girder/fix-default-s3-service-ansible
- [ ] #2661 from girder/fuse-cleanup-after-error
- [ ] #2473 from girder/delete-security-dialog-box
- [ ] #2663 from girder/pin-python-dateutil-for-botocore
- [ ] #2598 from girder/dicom-error
- [ ] #2599 from girder/assetstore-adapters-from-other-collections
- [ ] #2592 from girder/es6-arrow
- [ ] #2593 from girder/search-pending-icon
- [ ] #2591 from girder/pin-vega
- [ ] #2597 from girder/unpin-vega
- [ ] #2594 from girder/search-results
- [ ] #2595 from girder/dicom-timeout
- [ ] #2501 from wphicks/patch-1
- [ ] #2500 from girder/circle-v2
- [ ] #2502 from girder/pytest-ctest-test
- [ ] #2505 from girder/changelog-entries
- [ ] #2507 from girder/screenshots-in-circle
- [ ] #2506 from girder/libsasl2-dev_install_docs
- [ ] #2508 from girder/pytest-ports
- [ ] #2619 from girder/pydicom-pin
- [ ] #2458 from girder/hashsum-doc-lookup
- [ ] #2468 from girder/s3-range-request
- [ ] #2610 from girder/fix-item-breadcrumb-typo
- [ ] #2464 from girder/fix-model-name
- [ ] #2615 from girder/client-methods
- [ ] #2617 from girder/import-dicom
- [ ] #2616 from girder/unify-cli-commands
- [ ] #2512 from girder/circleci-docker
- [ ] #2511 from girder/pytest-test-plugins-behavior
- [ ] #2514 from girder/run-pytest-explicitly-in-ci
- [ ] #2518 from girder/circleci-deploy
- [ ] #2519 from girder/requirements
- [ ] #2608 from girder/swagger-url
- [ ] #2609 from girder/mongomock-pypi
- [ ] #2499 from girder/cherrypy-options-route
- [ ] #2478 from girder/candela-plugin-docs
- [ ] #2602 from girder/celery-cancel
- [ ] #2600 from girder/hidden-folder-behavior
- [ ] #2450 from girder/search-in-dicom-meta

**Comment by mgrauer** [comment]
I removed the release prep helper script, it didn't deserve to go in. Here's a [gist](https://gist.github.com/mgrauer/6703b75453eaf3c6de15f92d044bf78d).

**Comment by mgrauer** [comment]
This is good to go from my perspective.

**Comment by zachmullen** [comment]
I'll merge once it gets an approval

**Comment by manthey** [comment]
Should the package-lock file get updated as part of this, too?

File: CHANGELOG.rst [hunk 1]
```diff
@@ -32,38 +35,103 @@ Added Features
   (`#2596 <https://github.com/girder/girder/pull/2596>`_)
 * Added the server FUSE plugin that mounts Girder files in a read-only
   user-space filesystem. (`#2521 <https://github.com/girder/girder/pull/2521>`_)
+* Plugins can be enabled for Pytest. (`#2634 <https://github.com/girder/girder/pull/2634>`_)
```
Comment (manthey) [inline CHANGELOG.rst]: Should we move this to a subelement of the pytest entry (below)?  And, should this and other test-related features get moved to the Testing section?

Comment (zachmullen) [reply CHANGELOG.rst]: Yeah, could you add a commit?

**Comment by zachmullen** [comment]
@mgrauer could you approve? If so I'll tag 79fbbb5 as v2.5.0 and push it to pypi.

**Pull request merged**